care/emr/api/otp_viewsets/login.py

```
@@ -47,42 +50,64 @@ class OTPLoginView(EMRBaseViewSet):
+            random_otp = ""
+            if settings.USE_SMS:
+                random_otp = generate_otp(settings.OTP_LENGTH)
+                try:
+                    content = get_sms_content(
+                        settings.OTP_SMS_TEMPLATE_PATH, {"random_otp": random_otp}
+                    )
+                    sms.send_text_message(
+                        content=content,
+                        recipients=[data.phone_number],
+                    )
+                except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)